github4s/src/main/scala/github4s/interpreters/IssuesInterpreter.scala

```diff
@@ -155,6 +157,7 @@ class IssuesInterpreter[F[_]](implicit client: HttpClient[F], accessToken: Optio
       owner: String,
       repo: String,
       number: Int,
+      pagination: Option[Pagination] = None,
```

Comment:
The `pagination` parameter was added but not used anywhere. That is a problem in itself, but it also indicates another issue: it isn't being tested.